ci-operator/step-registry/openstack/provision/bastionproxy/openstack-provision-bastionproxy-commands.sh

```diff
@@ -139,25 +130,38 @@ if ! retry 60 5 $SSH_CMD uname -a; then
     openstack console log show ${server_id}
 		exit 1
 fi
-SQUID_IP=$bastion_fip
-
-echo "Deploying squid on $SQUID_IP"
 
 PASSWORD="$(uuidgen | sha256sum | cut -b -32)"
 SQUID_AUTH="${CLUSTER_NAME}:${PASSWORD}"
+echo ${SQUID_AUTH}>${SHARED_DIR}/SQUID_AUTH
+
+SQUID_IP=$bastion_fip
+if [[ "${CONFIG_TYPE}" == "proxy" ]]; then
+  PROXY_INTERFACE="$(openstack port list --network $MACHINES_NET_ID --server "$server_id" \
```

Comment:
This assumes there will only ever be one interface for the server on the machines network. If we don't make it more robust, we should at least add a comment explaining our assumption.

Reply:
I don't see why the bastion would have > 1 port connected on the machines network as of now, but you're right, this can evolve in the future; so we should document it here.